src/plugins/cascade/src/main/js/core/Utils.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Utils.js
+ *
+ */
+define(
+  'tinymce.plugins.cascade.core.Utils',
```

Comment:
Bolt configuration is pretty particular about the file hierarchy format...

Reply:
This `Utils` plugin doesn't get compiled and resulting files copied over to Cascade like for `image` & `link` & seems to work. I'm assuming that this plugin, like the regular `Settings` and `Utils` in the image plugin, just get compiled in the image & link plugins.

Reply:
Yeah, I think this is fine. I recall looking through WordPress' customizations at one point and they have a similar setup with some specific plugins.